ideainspect.groovy

```diff
@@ -242,15 +239,38 @@ private File findIdeaExecutable(OptionAccessor cliOpts) {
   ideaExecutable
 }
 
-private analyzeResult(File resultPath, List<String> acceptedLeves,
-                      List skipResults, List skipIssueFilesRegex) {
+private modifyIdeaProps(OptionAccessor cliOpts){
+  println " "
+  println "#"
+  println "# Modifying idea.properties for scope: $cliOpts.sc"
+
+  def ideaHome = cliOpts.i ?: (System.getenv("IDEA_HOME") ?: "idea")
+  def libraryHome = cliOpts.li.replace("~", System.getProperty("user.home"))
+  def ideaPropsFile = new File(libraryHome + "/idea.properties")
+  def outputLines = new ArrayList<String>();
+  assertPath(ideaPropsFile, "Idea Properties File",
+              "Use the `lib` proeprty in `.ideainspect` or the `-li` command line option to point\n" +
+                      "to a valid preferences folder.")
+  def lines = ideaPropsFile.text.split(System.getProperty("line.separator"))
+  for(line in lines){
+    if(!line.contains("idea.analyze.scope")){
+      outputLines.add(line);
+    }
+  }
+  outputLines.add("idea.analyze.scope=" + cliOpts.sc)
```

Comment:
I'd add a few additional comment lines explaining the idea & intention of this property and that it was generated by this tool. I.e. sth. like this

````

# idea-cli-inspector: GENERATED START
#   IDEA currently does currently not allow to pass the desired inspection scope as program parameter
#   but only as property. To reflect a command-line based inspection run this entry was updated/added. 
#   NOTE: This property only affect command line based inspection runs
idea.analyze.scope=
# idea-cli-inspector: GENERATED END
````